repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/filter/AbstractConfiguredFilterTest.scala

```diff
@@ -143,6 +167,18 @@ class AbstractConfiguredFilterTest
 
       filter.getInitialized shouldBe true
     }
+
+    it("should call doConfigurationUpdated") {
```

Comment:
Legit. I added a test for that specific case. Would you also like tests for the ordering of the other methods?